src/pages/myGeotab/whatsNew.tsx

```diff
@@ -1,8 +1,3045 @@
+		</ul>
+		<InformationalBox>
+			<p><b>!IMPORTANT:</b> Updating your application to version 10.0 may not work if an ObjectModel class is inherited.</p>
+		</InformationalBox>
+		<h2>TextMessage</h2>
+		<ul>
+			<li>
+				Added PropertySelector (Beta) support for the TextMessage type.
+			</li>
+		</ul>
+		<h2>IoxAddOn</h2>
+		<ul>
+			<li>
+				Added PropertySelector (Beta) support for the IoxAddOn type.
+			</li>
+		</ul>
+		<h2>IoxAddOnStatus</h2>
+		<ul>
+			<li>Added PropertySelector (Beta) support for the IoxAddOnStatus type.</li>
+		</ul>
+		<h2>DriverChange</h2>
+		<ul>
+			<li>
+				Updated the description of DriverChange DateTime to notify users that the DateTime must not be in the future when adding a DriverChange through API.
+			</li>
+		</ul>
+		<h2>DriverWhiteListContent</h2>
+		<ul>
+			<li>
+				DriverWhiteListContent is replaced by <b>DriverAuthListContent</b>
+			</li>
+			<li>
+				Prior to the 10.0 release, DriverWhiteListContent and DriverAuthListContent were both available to allow users to adjust to the transition.
+			</li>
+		</ul>
+		<h2>Condition And ConditionType</h2>
+		<ul>
+			<li>
+				Added the Group exception rule condition. This separates conditions in a single rule when different groups require different conditions.
+			</li>
+		</ul>
+		<h2>Geotab Developers</h2>
+		<ul>
+			<li>
+				Added a section to the <a href="https://geotab.github.io/sdk/software/guides/developing-addins/">Developing Add-ins</a> page suggesting to use the generator-addin repository.
+			</li>
+		</ul>
+		<h2>SDK Samples</h2>
+		<ul>
+			<li>
+				Added JavaScript samples to refresh the user interface.
+			</li>
+		</ul>
+		<h2>SDK Add-In Samples</h2>
+		<ul>
+			<li>Added Add-In samples for the AddInData API to the <a href="https://github.com/Geotab/sdk-addin-samples">Geotab/sdk-addin-samples</a> Github repository.</li>
+		</ul>
+	</div>
+);
+
+//ToDo Update Concepts Page Link
+const update9: ReactNode = (
+	<div className="paragraph">
+		<h2>Custom Security Identifiers For Add-Ins</h2>
+		<p>Custom security identifiers for MyGeotab Add-ins are now available. Identifiers can be defined in the configuration.json file of Add-ins, which will then add them to the list of permissions available when editing clearances. These definitions can support multiple languages.</p>
+		<p>Administrator clearance will remain non-editable and have all custom clearances enabled by default. All other default/system clearance levels only allow editing of custom security identifiers, while keeping the system defined default identifiers non-editable.</p>
+		<p><b>* NOTE</b>: By default, once the <code className="small-code-name">enableViewSecurityId</code> property in the configuration.json for an Add-in is set to True, a View “Add-in name” add-in identifier is created that must be enabled for users to be able to view the Add-in. More granular control needs to be set within the Add-in code for any custom identifiers added to the configuration.json.</p>
```

Comment:
Content inside of <code> tags should be surrounded by curly brackets & ticks { `` }